src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -2,16 +2,32 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using System.IO;
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    ///     Represents a directory
```

Comment:
Resharper made me do it. I actually like the leading space...it's more readable. The space is always stripped in Intellisense and doc gen, so shouldn't really matter.